module-services/service-appmgr/model/ApplicationManagerCommon.cpp

```diff
@@ -560,8 +560,9 @@ namespace app::manager
                                        (actionParams && actionParams->disableAppClose));
         SwitchRequest switchRequest(
             service::name::appmgr, targetApp->name(), targetApp->switchWindow, std::move(targetApp->switchData));
-        return handleSwitchApplication(&switchRequest, focusedAppClose) ? ActionProcessStatus::Accepted
-                                                                        : ActionProcessStatus::Skipped;
+        handleSwitchApplication(&switchRequest, focusedAppClose);
```

Comment:
why do we ignore app switch result now?

Reply:
first we check if the application is `ACTIVE_FORGROUND` or `ACTIVE_BACKGROUND` and then we give `ActionProcessStatus::Accepted`. My analysis shows that when we call application switching via `handleSwitchApplication`, we cannot give `ActionProcessStatus::Accepted` because the action will be called too soon - before the application starts.

Reply:
ok, it might be nice to add state, but can be cumbersone at the moment